### PR TITLE
chore: use tomli

### PR DESCRIPTION
**Pull request by henryiii**
Little bit of modernization.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -255,7 +255,7 @@ extend-exclude = [
 
 [tool.ruff.lint]
 extend-select = [
```
Comment (agoose77) [inline pyproject.toml:257]: @henryiii how did you know to remove this rule? B904? Curious as to your rationale.

Comment (henryiii) [reply pyproject.toml:257]: It doesn't do anything, B includes B904.

Comment (henryiii) [reply pyproject.toml:257]: (unlike Flake8, which needed B9)

Comment (agoose77) [reply pyproject.toml:257]: Gah, I misread the file and didn't see the wide selector (B). Thanks!

**Pull request merged**